src/main/java/stsjorbsmod/cards/wanderer/Prepare.java

```diff
@@ -51,19 +51,21 @@ public String getRawDynamicDescriptionSuffix() {
 
     @Override
     public void use(AbstractPlayer p, AbstractMonster m) {
+        addToBot(new GainBlockAction(p, p, block));
         addToBot(new ApplyPowerAction(p, p, new CoilPower(p, metaMagicNumber)));
 
+        // TODO: clean up later if we keep block -> gain coil
```

Comment:
It's okay to just remove the obsolete code; if we need it back later, we have version control for that :)